data/org.freedesktop.portal.Background.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0"?>
+<!--
+ Copyright (C) 2019 Red Hat, Inc.
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library. If not, see <http://www.gnu.org/licenses/>.
+
+ Author: Matthias Clasen <mclasen@redhat.com>
+-->
+
+<node name="/" xmlns:doc="http://www.freedesktop.org/dbus/1.0/doc.dtd">
+  <!--
+      org.freedesktop.portal.Background:
+      @short_description: Portal for requesting autostart and background activity
+
+      This simple interface lets sandboxed applications request that the
+      application is allowed to run in the background. If the application installs
+      an autostart file, it will be used to run the application when the user
+      logs in.
```

Comment:
The reference to autostart file here is a bit outdated, it should not say "installs an autostart file" but that is not true anymore. It should say something like "also allows applications to enable automatically starting on login".